src/main/resources/swagger-static/swagger-ui.js

```diff
@@ -4601,7 +4601,12 @@ Operation.prototype.encodeQueryCollection = function (type, name, value) {
   var encoded = '';
   var i;
 
-  if (type === 'default' || type === 'multi') {
+  if (name = 'x-custom-params') {
```

Comment:
I'm not sure, as I'm not a JS ninja, but it looks like an assignment and not a comparison :)

Shouldn't you use two or even three equal characters ?
Like: `==` or `===` ?